modules/phase_field/src/ics/SmoothCircleBaseIC.C

```diff
@@ -134,6 +138,5 @@ SmoothCircleBaseIC::computeCircleGradient(const Point & p, const Point & center,
   if (dist != 0.0)
     return _mesh.minPeriodicVector(_var.number(), center, p) * (DvalueDr / dist);
   else
-    return Gradient(0.0, 0.0, 0.0);
-
+    return 0.0;
```

Comment:
The old code mixed Point, Gradient, and RealGradient. Constructing a zero object to copy it is less clear (and more cluttered) than just returning zero in my opinion.